src/test/java/com/example/app/ReadableApiTest.java

```diff
@@ -63,6 +63,7 @@ void allLevelsWork() {
         logCapture.assertLoggedInOrder(
                 trace("first trace"),
                 trace("second trace"));
+        logCapture.assertNothingMatchingLogged(warn("first error"));
```

Comment:
I'd suggest assert... without the verb "matching" because it does not fit the positive case (assertLogged). So what about `assertNotLogged(...)`?